src/agent/agent.js

```diff
@@ -84,6 +84,12 @@ export class Agent {
 
                 this.bot.on('login', () => {
                     console.log('Logged in!');
+                    
+                    // Set skin for profile, requires Fabric Tailor. (https://modrinth.com/mod/fabrictailor)
+                    if (this.prompter.profile.skin)
+                        this.bot.chat(`/skin set URL ${this.prompter.profile.skin.model} ${this.prompter.profile.skin.path}`);
+                    else
+                        this.bot.chat(`/skin clear`);
```

Comment:
won't this always call `/skin clear` even if you don't have the fabric taylor?

Reply:
Yes. I suppose it can be done away since it's for a niche use-case where the agent has previously set a skin, and now re-joining without a skin defined in its profile. If not called, the agent will remain using the skin it previously had assigned since its cached on the player client.

Reply:
well I was worried about it causing issues in games without the fabric tailor. but it seems to be fine, the command just fails silently. looks good then.